source/templates.tex

```diff
@@ -3377,11 +3358,11 @@
 but will be checked when substituting into the partial specialization.
 \end{note}
 
-\rSec3[temp.class.spec.match]{Matching of class template partial specializations}
+\rSec3[temp.class.spec.match]{Matching of partial specializations}
 
 \pnum
-When a class template is used in a context that requires an instantiation of
-the class,
+When a template is used in a context that requires an instantiation of
+the template,
 it is necessary to determine whether the instantiation is to be generated
 using the primary template or one of the partial specializations.
 This is done by matching the template arguments of the class template
```

Comment:
Missed edit: "of the class template" -> "of the template"

Reply:
fixed